### PR TITLE
[RFR] Appliance modularize approach

### PR DESCRIPTION
**Pull request by mfalesni**

### CODE REVIEW

**Comment by RonnyPfannschmidt** [comment]
this gives me some structural ideas, we should have a deeper discussion

File: utils/appliance/__init__.py [hunk 1]
```diff
@@ -161,6 +162,8 @@ class IPAppliance(object):
     """
     _nav_steps = {}
 
+    evmserverd = ApplianceService.on_class('evmserverd')
```
Comment (RonnyPfannschmidt) [inline utils/appliance/__init__.py]: an idea i just had for declaring,
```
evmserverd = ApplianceSystemdService.declare(unit='evmserverd')
```

**Comment by dajoRH** [comment]
Lint report for commit 196cbb5f7fa3be86d0c2bb75bdf0bb66f2c1b521:

`utils/appliance/plugin.py`:
- :red_circle: Line 3:1: **F401** *'importlib' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: utils/appliance/plugin.py [hunk 2]
```diff
@@ -32,5 +31,5 @@ def logger(self):
         return logger
 
     @classmethod
-    def declare(cls, *args, **kwargs):
-        return AppliancePluginDescriptor(cls, args, kwargs)
+    def declare(cls, **kwargs):
+        return AppliancePluginDescriptor(cls, (), kwargs)
```
Comment (RonnyPfannschmidt) [inline utils/appliance/plugin.py]: crafty choice, this makes it hard to make badly readable declaration :) 🥇

**Pull request merged**